website/docs/r/schema_template_external_epg_subnet.html.markdown

```diff
@@ -13,15 +13,17 @@ Manages MSO Schema Template External EPG Subnet.
 ## Example Usage ##
 
 ```hcl
+
 resource "mso_schema_template_external_epg_subnet" "subnet1" {
-  schema_id = "5ea809672c00003bc40a2799"
-  template_name = "Template1"
-  external_epg_name =  "UntitledExternalEPG1"
-  ip = "10.101.100.0/0"
-  name = "sddfgbany"
-  scope = ["shared-rtctrl", "export-rtctrl"]
-  aggregate = ["shared-rtctrl", "export-rtctrl"]
+  schema_id         = mso_schema.schema1.id
+  template_name     = "Template1"
+  external_epg_name = "UntitledExternalEPG1"
+  ip                = "10.101.100.0/0"
```

Comment:
don't put /0 if you put a specific address put something like /24

Reply:
ok